src/main/res/menu/activity_conversations_wipe.xml

```diff
@@ -0,0 +1,15 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/select_all"
+        android:title="@string/select_all"
+        app:showAsAction="always"
+        android:orderInCategory="10"
```

Comment:
I think `android:orderInCategory` attribute value should be lesser than  value of `orderInCategory` of `id/delete_button`. Because I think we always want `select_all` should be at left side and `delete_button` should be at right side

Reply:
done